### PR TITLE
Menu final touches

### PR DESCRIPTION
**Pull request by micahblasko**

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -1,171 +1,427 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
```
Comment (StevonShaw) [inline html/index.html:4]: Can you turn your linter off?

File: html/index.html [hunk 2]
```diff
@@ -1,171 +1,427 @@
...
-    <link rel="stylesheet" href="../CSS/menu.css">
-</head>
-<body id="menuBody">
-    <p id="gamepadStatus">
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl"
+      crossorigin="anonymous"
+    />
```
Comment (StevonShaw) [inline html/index.html:13]: Another linter, auto config. 
Disable pls. It's obscuring your changes.

File: src/gamepad.js [hunk 3]
```diff
@@ -41,88 +41,87 @@ let controllerWarning = true;
...
 
-if(controllerWarning === true){
-	gamepadWarning.textContent = "Please Press a Button or\r\nConnect a Gamepad";
+if (controllerWarning === true) {
+  gamepadWarning.textContent = "Please Press a Button or\r\nConnect a Gamepad";
 }
 
 // Upon controller detection assign abstracted controls appropriately
 window.addEventListener("gamepadconnected", (e) => {
-	controllerWarning = false;
```
Comment (StevonShaw) [inline src/gamepad.js:50]: Again, linter is over-aggressive and adding too many changes at once.

**Review by StevonShaw — CHANGES_REQUESTED** [review]

File: html/index.html (hunk 1 above)
Comment (StevonShaw) [reply html/index.html:4]: If you take a look at the changes of this file, nearly every line is highlighted with whitespace, or terminating characters.

This causes your changes to be lost in the mix of all these other lines.

File: html/index.html [hunk 4]
```diff
@@ -1,171 +1,427 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (StevonShaw) [inline html/index.html:6]: Can you turn your linter off?
If you take a look at the changes of this file, nearly every line is highlighted with whitespace, or terminating characters.

This causes your changes to be lost in the mix of all these other lines.

File: html/index.html (hunk 2 above)
Comment (StevonShaw) [inline html/index.html:13]: Another linter, auto config.
Here it's spreading a button element's attributes out to multiple lines.
We don't need this, and it makes it harder to find elements in this file.

File: src/gamepad.js [hunk 5]
```diff
@@ -41,88 +41,87 @@ let controllerWarning = true;
...
+    options = 7;
+    //TODO: sort out Dpad for steering wheel
+    lsX = 0;
+    r2 = 1;
+    l2 = 2;
+
+    handleDirection = handleDirectionSteeringWheel;
+    lookHorizontal = lookHorizontalSteeringWheel;
+  }
+  chooseControlsEventId = setInterval(chooseControls, 100);
```
Comment (StevonShaw) [inline src/gamepad.js:102]: Again, linter is over-aggressive and adding too many changes at once.

**Review by StevonShaw — CHANGES_REQUESTED** [review]

**Comment by StevonShaw** [comment]
I'm sure you didn't mean to modify nearly every line with a 'touch-up' merge.
And it appears you have a IDE setting turned on that everyone else does not.

**Comment by micahblasko** [comment]
Yeah I have no idea why I have had the same prettier auto thing this whole time and I have saved these files before but yeah, ill just revert all the changes and make them again without the auto formatting

**Pull request closed without merging**